core/actionProxy/Dockerfile

```diff
@@ -1,13 +1,13 @@
 # Dockerfile for docker skeleton (useful for running blackbox binaries, scripts, or python actions).
-FROM python:2.7.12-alpine
+FROM python:2.7.13-alpine
 
 # Upgrade and install basic Python dependencies
 RUN apk add --no-cache bash \
  && apk add --no-cache --virtual .build-deps \
         bzip2-dev \
         gcc \
         libc-dev \
-  && pip install --no-cache-dir gevent==1.1.2 flask==0.11.1 \
+  && pip install --no-cache-dir gevent==1.2.1 flask==0.12 \
```

Comment:
The official Python 2 docker image does not contain Python 3 and vice versa so I had to create a [mind-meld between the two](https://github.com/docker-library/python/issues/178).  So`dockerskeleton`is now a single Docker image with both Python 2.7.13 and Python 3.6.0 available.  We need to be explicit about using python2 vs. python3 as well as pip2 vs. pip3.